docs/js/badge.js

```diff
@@ -0,0 +1,16 @@
+/**Badge JS**/
+
+const badgeBtn = document.querySelector('#badgeBtn')
+const hideBadge = document.querySelector('#hideBadge')
+
+badgeBtn.addEventListener('click', checkBadge)
+
+function checkBadge() {
+  if (hideBadge.classList.contains('badge-content')) {
+    hideBadge.classList.remove('badge-content', 'position-absolute', 'sm-badge')
+    hideBadge.classList.add('hidden')
+  } else {
+    hideBadge.classList.remove('hidden')
+    hideBadge.classList.add('badge-content', 'position-absolute', 'sm-badge')
+  }

```

Comment:
You can use `hideBadge.classList.toggle('hidden')`, it would work the same as the whole function `checkBadge`.

Reply:
Ohh okay didn't know this. Will change it.